code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -574,8 +629,10 @@
 		disarm()
 		return
 	if(is_station_level(bomb_location.z))
-		var/datum/round_event_control/beer_clog/beer_event = new()
```

Comment:
is this round event removed?

Reply:
Seems like it

Reply:
No, should it be? 
I thought I should leave it for anyone, like admins, who wanted to use it.

Reply:
Yeah I think I was asking because you removed this code here, why is this removed here. i don't know

Reply:
Because it spawns liquids instead, like it should